packages/client/mutations/UpdatePokerScopeMutation.ts

```diff
@@ -113,17 +113,6 @@ const UpdatePokerScopeMutation: StandardMutation<TUpdatePokerScopeMutation, Hand
   return commitMutation<TUpdatePokerScopeMutation>(atmosphere, {
     mutation,
     variables,
-    updater: (store) => {
-      const payload = store.getRootField('updatePokerScope')
-      const meeting = payload.getLinkedRecord('meeting')
-      const newStages = payload.getLinkedRecords('newStages')
-      if (!meeting || !newStages) return
-      const phases = meeting.getLinkedRecords('phases')
-      const estimatePhase = phases.find((phase) => phase.getType() === 'EstimatePhase')!
-      const stages = estimatePhase.getLinkedRecords('stages')
-      const nextStages = [...stages, ...newStages]
```

Comment:
When selecting all issues, `stages` will include many of the same issues as `newStages` so it will duplicate the issues. 

We could make sure that `nextStages` only contains unique stages, but it looks like the `updater` here isn't very helpful so I'd prefer to remove it. I've tested updating the poker scope with and without the updater with two users by selecting & deselecting all as well as selecting & deselecting individual issues. The behaviour is identical either way.

Reply:
i tested and it works. out of curiosity, do we where the relay store is updated so it actually shows selected/deselected issues without this updater? i suspect it'd be a meeting subscription. i just want to make sure we don't rely on some accidental side effect

Reply:
I'm not sure. It might be because the local data update isn't very complex, but I'd like to get the Maintainer's input on this

Reply:
Would love to look into the actual implementation, but basically because the objects in "stages" and "newStages" have the same id they are treated as identical, i.e. merged in the store.